pkg/model/dev_manifest.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2021 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package model
+
+import "github.com/okteto/okteto/pkg/log"
+
+//DevManifest represents an okteto manifest
+type DevManifest struct {
+	Name      string          `json:"name" yaml:"name"`
```

Comment:
Remove `Name`

Reply:
In the spec it's mandatory to put the name on the manifest top level

Reply:
yes, but that's to autocreate the namespace. but we are not implementing that yet